modules/number-picker/main/index.coffee

```diff
@@ -72,27 +72,33 @@ class NumberPicker extends hx.EventEmitter
     if @options.max isnt undefined then @max @options.max
     if @options.min isnt undefined then @min @options.min
     if @options.disabled then @disabled(@options.disabled)
-
-    @selectInput
-      .attr('data-value', @options.value)
-      .value(@options.value)
+    @value @options.value
```

Comment:
We were setting the value from the constructor using the input which bypasses the checks inside the value method.
We may want to suppress the event here although you can't listen to it before this is fired so I'd say this is probably fine.